### PR TITLE
Varya: Fix image alingment issue and code tidying.

### PR DESCRIPTION
**Pull request by allancole**
This PR fixes the alignment issues with wide and full images in the editor.

Before|After
------------|------------
imageimage|imageimage

Fixes: #103

### CODE REVIEW

File: varya/assets/sass/blocks/utilities/_editor.scss [hunk 1]
```diff
@@ -158,6 +165,21 @@
 }
 
 /* Block Alignments */
+.wp-block {
+
+	// Gutenberg injects a rule that limits the max width of .wp-block to 580px
+	// This line overrides it to use the responsive spacing rules for default width content
+	&:not(.alignwide):not(.alignfull):not([data-align="full"]):not([data-align="wide"]){
```
Comment (kjellr) [inline varya/assets/sass/blocks/utilities/_editor.scss:172]: Ooof. Is there any way to fix this without all of those `:not()` selectors?

Comment (jffng) [reply varya/assets/sass/blocks/utilities/_editor.scss]: I pushed a change that I believe fixes the issue, without using the `:not()` selectors.

File: varya/assets/sass/blocks/utilities/_editor.scss [hunk 2]
```diff
@@ -158,6 +165,24 @@
 }
 
 /* Block Alignments */
+.wp-block {
+
+	// Gutenberg injects a rule that limits the max width of .wp-block to 580px
+	// This line overrides it to use the responsive spacing rules for default width content
+	max-width: var(--responsive--aligndefault-width);
```
Comment (jffng) [inline varya/assets/sass/blocks/utilities/_editor.scss:172]: This rule will apply to every block in the editor.

File: varya/assets/sass/blocks/button/_editor.scss [hunk 3]
```diff
@@ -30,3 +30,7 @@
 		border-radius: 0;
 	}
 }
+
+div[data-type="core/button"] {
```
Comment (kjellr) [inline varya/assets/sass/blocks/button/_editor.scss:34]: Is this meant to be part of this PR?

Comment (kjellr) [reply varya/assets/sass/blocks/button/_editor.scss:34]: Oh wait, I see — it's been moved. Nevermind. 👍

**Pull request merged**